packages/@coorpacademy-components/src/organism/review-skills/test/fixtures/loading.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  props: {
+    'aria-label': 'Review Skills Container',
+    title: 'Skills you can revise',
+    isLoading: true,
+    isLoadingAriaLabel: 'Review skills container is loading'
```

Comment:
vue qu'il y a un état loading, donc soit on laisse listSkills la possibilité d'être undefined (il faudrait supprimer l'isRequired pour listSkills )
```suggestion
    isLoadingAriaLabel: 'Review skills container is loading',
    listSkills: undefined
```